src/twin/system_info.rs

```diff
@@ -24,10 +75,18 @@ lazy_static! {
 pub struct SystemInfo {
     #[serde(skip_serializing)]
     tx_reported_properties: Option<mpsc::Sender<serde_json::Value>>,
+    #[serde(skip_serializing)]
+    tx_outgoing_message: Option<mpsc::Sender<IotMessage>>,
     os: serde_json::Value,
     azure_sdk_version: String,
     omnect_device_service_version: String,
     boot_time: Option<String>,
```

Comment:
imho this should go into a new struct, e.g. `Info`. Afterwards you can remove all `#[serde(skip_serializing)]` and only serialize the structs you want to serialize

Reply:
still open, work on it...

Reply:
done